src/shared.ts

```diff
@@ -385,3 +385,7 @@ export function isNodeModulesResolution(args: esbuild.OnResolveArgs) {
       isInNodeModules(args.importer))
   );
 }
+
+export const EXTENSION_FILTER = /\.(ts|tsx|js|jsx|mts|mjs|json|wasm)$/;
+export const DATA_FILTER =
+  /^(application|text)\/(typescript|javascript|x-typescript|x-javascript|ecmascript|jsx|tsx|json|wasm|octet-stream|node)/;
```

Comment:
I believe these are all the file types and data types that the deno loader plugin support. I updated it so that onLoad will only be called for those types of file and data imports.